### ISSUE TITLE
CycleGAN error in models.py

### ISSUE THREAD

**Issue by cyduoot** [eriklindernoren/PyTorch-GAN#128]
In the class `GeneratorResNet`, in line 49 and line 82, the parameter of `nn.ReflectionPad2d` should be namely`3` but not `channels`. It should be decided by the kernel size but not the channels.

**Comment by Boyden** [comment]
Yeah, It should be 3 to keep the same shape after using `nn.Conv2d(channels, out_features, 7)`

**Comment by Healingl** [comment]
> In the class `GeneratorResNet`, in line 49 and line 82, the parameter of `nn.ReflectionPad2d` should be namely`3` but not `channels`. It should be decided by the kernel size but not the channels.

This change is efficient and correct, especially for multi-channels images (e.g. 1x256x256 or 2x256x256).
